website/src/content/docs/docs/howtos/Generate client libraries/04renaming.mdx

```diff
@@ -270,6 +272,8 @@ public final class PetStoreNamespaceClient {
 </TabItem>
 </Tabs>
 
+You cannot at this moment rename paramters in the client.tsp file.
```

Comment:
I'd add more to this and show an example of having the `@clientName` decorator over a parameter in an example operation. And say:
```suggestion
You cannot at this moment rename parameters in the client.tsp file. You will need to add the `@clientName` decorator over the parameter directly, example:

```